icubam/db/synchronizer.py

```diff
@@ -4,12 +4,24 @@
 from typing import TextIO
 
 import pandas as pd
+import json
 from absl import logging
 from dateutil import tz
 
 from icubam.db import store
 
-ICU_COLUMNS = ['name', 'region', 'dept', 'city', 'lat', 'long', 'telephone']
+ICU_DTYPE = {
```

Comment:
Maybe just keep `ICU_COLUMNS` and make it a dict. Iterating over it will return keys as before.

Reply:
I tried initially, but we do other things than iterating over it IIRC, so I find it safer this way.

Reply:
all this information is available from store.py.  This is supposed to be the source of truth...
I would rather use as much store.py as possible to avoid duplicating the information and also avoiding things drifting appart.

Reply:
If you look closer, you'll see that the types are not the same. The point of this type dictionary is to prevent pandas from doing random conversions when reading the CSV. In particular, if a value is missing in a row, pandas inserts a value which depends on the type it has inferred for the column (`NA` or `nan`, for example). It also messes with GPS coordinates, trying to round them.

So to summarize, the real schema is indeed defined by `store.py`. What we want here is to prevent pandas from messing with the data it passes to the store. I'll add a comment.